### PR TITLE
SD-106: Update delivey email content

### PR DESCRIPTION
**Pull request by joehod**
https://collaboration.homeoffice.gov.uk/jira/browse/SD-106

This is a change requested by the BRP team to change the content on the lost-stoled card response email. The new content is specified within the ticket above.

### CODE REVIEW

File: services/email/templates/customer/plain/delivery.mus [hunk 1]
```diff
@@ -26,3 +22,4 @@ If you're unsure an email is from UKVI:
  * do not reply to it or click any links
 
 {{/data}}
+
```
Comment (sulthan-ahmed) [inline services/email/templates/customer/plain/delivery.mus:25]: Did you mean to add an extra line?

Comment (joehod) [reply services/email/templates/customer/plain/delivery.mus:25]: Ah no, think my linting thing told me to. Will remove that.

Comment (sulthan-ahmed) [reply services/email/templates/customer/plain/delivery.mus]: 👍

File: services/email/templates/customer/plain/delivery.mus [hunk 2]
```diff
@@ -5,16 +5,12 @@ We are currently experiencing very high volumes of requests
```
Comment (sulthan-ahmed) [inline services/email/templates/customer/plain/delivery.mus:5]: in the ticket it says they want this

`You have reported that your BRP hasn't arrived`

Is this already there in the heading?

Not a must but nice to have a screenshot of the change

Comment (joehod) [reply services/email/templates/customer/plain/delivery.mus:5]: Yeah, that is set in the heading. The email isn't coming through to my digital one.

**Review by sulthan-ahmed — APPROVED** [review]

**Comment by sulthan-ahmed** [comment]
Lgtm

**Pull request merged**